Makefile

```diff
@@ -30,6 +30,7 @@ OCM_CLIENT_ID := ${OCM_CLIENT_ID}
 OCM_CLIENT_SECRET := ${OCM_CLIENT_SECRET}
 ENABLE_AUTH := $(or ${ENABLE_AUTH},False)
 DELETE_PVC := $(or ${DELETE_PVC},False)
+REPLICAS_COUNT = $(shell if ! [ "${TARGET}" = "minikube" ];then echo 3; else echo $(or ${SERVICE_REPLICAS_COUNT},3);fi)
```

Comment:
why not make it 3 by default, is there a problem to run 3 replicas in minikube?

Reply:
it is 3 by default but if u have bad laptop u prefer to have an option to set replicas count

Reply:
yes but we run in jenkins with minikube as well. so maybe if you want to run it locally with replica one it should not be the default

Reply:
i was asked to make this default and i am ok with it. If service will run in production why not to test it the same way.

Reply:
why not just `REPLICAS_COUNT := $(or ${REPLICAS_COUNT},3)` ?

Reply:
explained but i can change it @romfreiman  ?